### DISCUSSION TITLE
Multi Brand & Multi Currency

### DISCUSSION THREAD

**Discussion by rudik99** [saleor/saleor#11806] [Q&A]
Hi there,

Just stumbled across this project. 
Have been playing with the demos, wondering if Saleor has the ability to do multi brand sites, with multi currency for each brand?
for example:

Brand A with AUD & USD
Brand B with USD & EUR

**Comment by timuric** [comment]
You can create channels for each currency for each brand. Then each storefront would be working with specific channels only.
Could you share more details about your use case? 
Which data/objects and settings you want to share between brands/sites and what do you want to keep separate?

**Reply by rudik99** [reply]
Thanks! 

Wanting to manage orders, warehouses, inventory as a single set , in a single backend.

But likely that most products will be unique to each brand

Some pages to be used across all, for example, contact us pages etc - lots of unique pages

Categories to be unique, but I think some collections may end up having the same name (like Sale) but assuming if a product isn't enabled for the store front its not going to show anyway?

We can have the same customers across each brand - so assuming the login will work across the lot.

Common cart isn't critical, but interesting if its possible.

We would use the same payment processor for each brand i.e stripe, but they'd have their own API KEYS and Merchant accounts - so seperate.
